src/modules/commander/Commander.cpp

```diff
@@ -3078,6 +3079,7 @@ Commander::run()
 			fd_status.fd_battery = _failure_detector.getStatusFlags().battery;
 			fd_status.fd_imbalanced_prop = _failure_detector.getStatusFlags().imbalanced_prop;
 			fd_status.imbalanced_prop_metric = _failure_detector.getImbalancedPropMetric();
+			fd_status.fd_mpc_vz = _failure_detector.getStatusFlags().mpc_vertical_rate;
```

Comment:
Consider renaming `fd_mpc_vz` to `fd_mpc_vertical_rate` in order to match  `failure_detector_status_u.mpc_vertical_rate`, like the other fields in `failure_detector_status_s`. (or the other way around).
That could remove some uncertainty about the fields being related or not when reading the failure detector code.

Reply:
Changed mpc_vertical_rate -> mpc_vz so it matches parameter names better as well.